birdhouse/components/scheduler/deploy_data_job.env

```diff
@@ -2,8 +2,26 @@
 # Generic job for deploy-data script.
 # Configuration vars, set in env.local before sourcing this file.
 # This job assume the "scheduler" component is enabled.
+#
+# This will write the deploy-data configuration to stdout. The calling script
+# should assign this to a variable that is used as a template variable in
+# config.yml.template. For example:
+#
+# . deploy_data_job.env
+# BIRDHOUSE_AUTODEPLOY_EXTRA_SCHEDULER_JOBS="
+# $BIRDHOUSE_AUTODEPLOY_EXTRA_SCHEDULER_JOBS
+# $(print_cron_config)
+# "
 ##############################################################################
 
+# WARNING!! Using this file to update BIRDHOUSE_AUTODEPLOY_EXTRA_SCHEDULER_JOBS is deprecated.
+#           Please convert any custom jobs that use this script to components. See the following
+#           components for examples on how to do this:
+#           - optional-components/scheduler-job-deploy_raven_testdata
+#           - optional-components/scheduler-job-deploy_xclim_testdata
```

Comment:
Since we have a new way using "components", can you revert this file to the previous state to avoid all the external existing jobs?  Here are some of our external jobs using `deploy-data`, we have more that are private: https://github.com/bird-house/birdhouse-deploy-ouranos/tree/master/scheduler-jobs

You can keep this warning.